### DISCUSSION TITLE
Facing 503 socket timeout issue during scaling of pods using HPA with CPU threshold of 70%

### DISCUSSION THREAD

**Discussion by parshva-shah-dsw** [istio/istio#49141] [Q&A]
@zirain @howardjohn   We have OLTP and Batch application pods running in single cluster group. This cluster group needs autoscale of batchpods during EOD. It has been observed that we encounter **503 socket connection timeout** error whenever a new worker node spins upto provision new pods. This creates service disruption in our production environment. Cluster is hosted on AWS - EKS v1.25 using Istio v1.18.5. 
Please let us know if this issue has some parameteric fix or required specific version of istio or eks.

**Comment by parshva-shah-dsw** [comment]
@Stono can you help here

**Comment by Stono** [comment]
Not sure why you've tagged me?

**Reply by parshva-shah-dsw** [reply]
@Stono similar issue was reported previously https://github.com/istio/istio/issues/7665 in istio 1.0 wanted to understand is there any configuration apart from retry mechanism in virtual service

**Reply by Stono** [reply]
that's 11 major releases ago!  Almost nothing from that will be true today.

I wrote a blog on debugging 503's https://karlstoney.com/istio-503s-ucs-and-tcp-fun-times/ but you need to start with understanding where they're coming from in the first place (istio logging, knsiff, etc).
